client/src/components/screens/PatientSubmission/components/SecondPage.tsx

```diff
@@ -15,6 +16,39 @@ interface Props {
   }
 }
 
+type Practitioner = {
+  data?: {
+    family?: string;
+    given?: string;
+    id?: string;
+    license?: string;
+  }
+  isValid: boolean;
+}
+
+const getPractitioner = (resource: any) : Practitioner => {
+  if(
+    resource?.name?.length > 0 &&
```

Comment:
you could skip the length test by doing resource?.name?[0]?.given?[0] and so forth